2007-09-09  Aurelien Jarno  <aurelien@aurel32.net>

	* iconvdata/gconv-modules: Add LATIN9 as an alias to 
	ISO-8859-15.

--- a/iconvdata/gconv-modules
+++ b/iconvdata/gconv-modules
@@ -395,6 +395,7 @@ alias	ISO885915//		ISO-8859-15//
 alias	ISO-IR-203//		ISO-8859-15//
 alias	ISO_8859-15//		ISO-8859-15//
 alias	LATIN-9//		ISO-8859-15//
+alias	LATIN9//		ISO-8859-15//
 alias	ISO_8859-15:1998//	ISO-8859-15//
 module	ISO-8859-15//		INTERNAL		ISO8859-15	1
 module	INTERNAL		ISO-8859-15//		ISO8859-15	1
